src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/TelemetryReporterTests.cs

```diff
@@ -15,7 +16,7 @@ public class TelemetryReporterTests
     [Fact]
     public void NoArgument()
     {
-        var reporter = new TestTelemetryReporter();
+        var reporter = new TestTelemetryReporter(new RazorLoggerFactory([]));
```

Comment:
IMO, just make the `TestTelemetryReporter` handle passing the empty factory to the base constructor

Reply:
Good idea.